clang/lib/CodeGen/CodeGenTypes.cpp

```diff
@@ -125,9 +125,100 @@ bool CodeGenTypes::isRecordLayoutComplete(const Type *Ty) const {
   return I != RecordDeclTypes.end() && !I->second->isOpaque();
 }
 
+#ifndef INTEL_SYCL_OPAQUEPOINTER_READY
```

Comment:
@jsji, it would be great to clarify the commit message the reason to revert the community patch. It's not obvious why do we need to keep these changes in our branch.

Reply:
It is not the first time we guard the revert , but sure, added in commit message.